components/Common/Badge/index.module.scss

```diff
@@ -0,0 +1,63 @@
+.wrapper {
+  @apply border rounded-full flex items-center w-fit py-1 pl-1 pr-2.5 dark:bg-neutral-900;
+
+  &Green {
+    @apply border-green-200 dark:border-green-700 bg-green-100;
+  }
+
+  &Warning {
+    @apply border-warning-200 dark:border-warning-700 bg-warning-100;
+  }
+
+  &Error {
+    @apply border-danger-200 dark:border-danger-700 bg-danger-100;
+  }
+}
+
+.badge {
+  @apply text-sm font-medium rounded-full border px-2.5 py-0.5 mr-2 text-white;
+
+  &Green {
```

Comment:
Also just a fyi, class names should not start with uppercase name

Reply:
With using `&` operator adding parent class name for this example; output is `.badgeGreen`. I updated with sub class as you mentioned above 👍

Reply:
I know what the & operator does. I'm saying you don't need to use it. This is CSS modules, we don't need to care much about uniqueness of class names ;)